openvalidators/reward/dpo.py

```diff
@@ -33,17 +33,23 @@ def name(self) -> str: return RewardModelType.dpo.value
     def __init__(self, device: str):
         super().__init__()
         self.device = device
+        self.penalty = 1.2
```

Comment:
do we have any data for why we picked 1.2 for penalty?

Reply:
It is the same default parameter used by huggingface and was retrieved from this paper (https://arxiv.org/pdf/2305.14314.pdf)

Reply:
I think that adding this reference to the code in one comment line could help clarify future doubts